site/content/entry/yuzu-progress-report-may-2023/index.md

```diff
@@ -0,0 +1,828 @@
+
+{{< imgs
+	"./aw.png| Bond, yuzu Bond. (Advance Wars 1+2: Re-Boot Camp)"
+  >}}
+
+Recent changes made to audio emulation improved performance and solved crashes, but also had an interesting regression.
+Some sound effects played at a slower speed.
+
+After doing some extensive reverse engineering, Maide reached the conclusion that due to how the DSP works, a {{< gh-hovercard "10221" "5 ms timeout" >}} is needed in the processing wait.
+With the timeout implemented, `SUPER MARIO ODYSSEY` and `Kirby Star Allies`, among others can now play back their audio at full speed.
+
+[danilaml](https://github.com/danilaml) continues to deliver the good stuff.
+This time, simply {{< gh-hovercard "10362" "updating cubeb" >}} (one of our audio backends, and most of the time the default one) fixed a bug that muted the emulator after resuming the PC from sleep.
+Feel free to close your lids and continue later!
+
+Users playing games with lots of shaders or with slow storage devices, like mechanical drives, will know that the audio can desync and sound crackled.
+bylaws suggested allowing the audio buffer to {{< gh-hovercard "10471" "wait indefinitely" >}} if a queue is too big, so Maide implemented this idea and et voilà, the problem is gone!
+(Pardon my French; Spanish is cooler).
+
+## UI improvements
+
+If your yuzu is configured with multiple user profiles, and the `Prompt for user on game boot` option in `Emulation > Configure > General` option is enabled, you will have to click the profile you want to boot and then click OK.
+Well, for the newcomer [Qigo42](https://github.com/Qigo42), this was unsatisfactory, so they allowed the {{< gh-hovercard "10189" "profile selection pop-up" >}} to work with just a double click!
+Nothing beats quality-of-life changes. Thank you!
+
+[jbeich](https://github.com/jbeich) is back, fixing {{< gh-hovercard "10205" "BSD support" >}} again!
+What a Chad. It’s always good to have more OS alternatives working.
+
+Continuing the trend in quality-of-life changes, newcomer [grimkor](https://github.com/grimkor) has a nice gift for us.
+
+{{< imgs
+	"./context.png| And now, for your convenience…"
+  >}}
+
+A {{< gh-hovercard "10352" "context menu" >}} for the filter and antialiasing options! You no longer will have to go through the entire list of options if you want to try a different one, just right click and choose.
+Thank you!
+
+And finally, an oversight leading to weird issues, one of those cases where we ask ourselves “what the hell is the user doing to trigger this?!”
+{{< gh-hovercard "10482" "context menu" >}}
+In german77’s words, “Qt solution to Qt problem.”
+The game list was still active in the background while a game was running, so under special circumstances, you could try to boot a second game, causing a colourful set of issues, leading to deafening sirens sounding off, havoc, mayhem, and too many mosquitoes in a closed space.
+Needless to say, disabling the game list when a game is selected solved the issue.
```

Comment:
```suggestion
If you double clicked a game in the game list, you could end up double clicking it again or pressing enter before the game list got unloaded, which would cause yuzu to try to load the game twice, and usually just crash. 
german77 fixed this up by {{< gh-hovercard "10482" "immediately disabling the game list" >}} after a game has been launched.
```